infra_tooling_account/infra_tooling_account/infra_tooling_common_stack.py

```diff
@@ -249,6 +251,16 @@ def create_notification_lambda(
             )
         )
 
+        notification_lambda_role.add_to_policy(
+            iam.PolicyStatement(
+                actions=["secretsmanager:GetSecretValue"],
+                effect=iam.Effect.ALLOW,
+                resources=[
+                    "*"
+                ],  # to be able to retrieve Secrets required for SMTP sender
+            )
```

Comment:
maybe, an idea for future - instead of allowing getsecretvalue for "*"
we can:
1) ask user put a specific tag on SMTP secrets
2) role will be able to read "*" resources, but only those which are also tagged "config" : "salmon", for example.
Let's discuss in the next sync-up

Reply:
sure, I support this idea, thank you!